app/scripts/modules/kubernetes/src/v2/manifest/wizard/manifestEntry.component.ts

```diff
@@ -6,17 +6,18 @@ import './manifestEntry.less'
 
 class KubernetesManifestCtrl implements IController {
   public command: IKubernetesManifestCommand;
+  public change() { }
```

Comment:
This is a function definition, not a type definition. Should be `public change: () => void;`.